test/testenv/verificationutils.go

```diff
@@ -489,3 +500,45 @@ func VerifySplunkServerConfSecrets(deployment *Deployment, testenvInstance *Test
 		}
 	}
 }
+
+// VerifyPVCExists verifies if PVC exist
+func VerifyPVCExists(deployment *Deployment, testenvInstance *TestEnv, ns string, pvcName string, expectedtoExist bool, verificationTimeout time.Duration) {
+	gomega.Eventually(func() bool {
+		result := false
+		pvcexists := false
+
+		pvcsList := DumpGetPvcs(testenvInstance.GetName())
+		for i := 0; i < len(pvcsList); i++ {
+			if strings.EqualFold(pvcsList[i], pvcName) {
+				pvcexists = true
+				break
+			}
+		}
+		if expectedtoExist && pvcexists || !expectedtoExist && !pvcexists {
+			result = true
+			logf.Log.Info("PVC status (exists or not) is as expected: ", "PVC", pvcName, "STATUS", pvcexists, "EXPECTED", expectedtoExist)
+		} else {
+			err := errors.New("incorrect PVC status (exists when it shouldn't or the opposite): ")
+			logf.Log.Error(err, "PVC", pvcName, "STATUS", pvcexists, "EXPECTED", expectedtoExist)
+
+		}
+		return result
+	}, verificationTimeout, PollInterval).Should(gomega.Equal(true))
+}
+
+// VerifyPVCsPerDeployment verifies for a given deployment if PVCs (etc and var) exists
+func VerifyPVCsPerDeployment(deployment *Deployment, testenvInstance *TestEnv, deploymentType string, expectedtoExist bool, verificationTimeout time.Duration) {
+	if deploymentType == "shc-deployer" || deploymentType == "cluster-master" {
+		PvcName := fmt.Sprintf(PVCString, "etc", deployment.GetName(), deploymentType, 0)
+		VerifyPVCExists(deployment, testenvInstance, testenvInstance.GetName(), PvcName, expectedtoExist, verificationTimeout)
+		PvcName = fmt.Sprintf(PVCString, "var", deployment.GetName(), deploymentType, 0)
+		VerifyPVCExists(deployment, testenvInstance, testenvInstance.GetName(), PvcName, expectedtoExist, verificationTimeout)
+	} else {
+		for i := 0; i <= 2; i++ {
```

Comment:
Will this work if deployment has more/less than 2. I think its not good to hardcode number of nodes here ?

Reply:
indeed